### PR TITLE
G0-5153_advanceRec: sales invoice advance receipt issue and tax control issue fixed

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html [hunk 1]
```diff
@@ -113,7 +113,8 @@ <h4 class="modal-title bg pull-left">Adjust Payment</h4>
                                 <div class="form-group clearfix">
                                     <label class="col-md-2">Amount <span class="text-danger">* </span></label>
                                     <div class="col-md-3">
-                                        <input type="text" class="form-control" required #tdsAmountBox name="tdsAmount.amountForAccount" (ngModelChange)="changeTdsAmount($event)" [(ngModel)]="adjustVoucherForm.tdsAmount.amountForAccount" placeholder="Amount">
+                                        <input type="text" class="form-control" required #tdsAmountBox decimalDigitsDirective [prefix]="currencySymbol" [mask]="inputMaskFormat" name="tdsAmount.amountForAccount" (ngModelChange)="changeTdsAmount($event)" [(ngModel)]="adjustVoucherForm.tdsAmount.amountForAccount"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html]: Why we are using object in name?

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
      */
     public onCancel(): void {
         if (this.adjustPayment && this.adjustPayment.totalAdjustedAmount && this.adjustPayment.grandTotal && this.adjustPayment.totalAdjustedAmount - this.adjustPayment.grandTotal > 0) {
-            this.toaster.warningToast('The adjusted amount of the linked invoice\'s is more than this receipt');
+            this.toaster.warningToast('The adjusted amount of the linked invoice\'s is more than this receipt due amount');
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts]: This same message is getting used multiple times in this file. Create a variable for this and use it .

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html (hunk 1 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html]: Name is required for json key here

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts (hunk 2 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts]: Both message is different please check

Comment (rohit-walkover) [reply apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts]: Both is different message please check

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html (hunk 1 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html]: we are using for JSON key

**Pull request merged**